### PR TITLE
add testing

### PR DESCRIPTION
**Pull request by alexgaal**

### CODE REVIEW

File: tests/CreateFileFromStubTest.php [hunk 1]
```diff
@@ -0,0 +1,207 @@
+<?php
+
+test('creates an action from stub', function () {
+    $fs = new \Illuminate\Filesystem\Filesystem();
+    $stub = 'action.stub';
+
+    $path = beyond_path() . "/tests/files/{$stub}";
```
Comment (regnerisch) [inline tests/CreateFileFromStubTest.php:7]: We should not depend on `beyond_path` here. Use `__DIR__` instead

File: tests/CreateFileFromStubTest.php [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+    beyond_copy_stub($stub, $path);
```
Comment (regnerisch) [inline tests/CreateFileFromStubTest.php:8]: We do not need to test if every stub is set as expected. I would test if `beyond_copy_stub` _does copy_ any stub as expected. Also we should test if all stubs we expect to be existing are existing.

**Review by regnerisch — COMMENTED** [review]
I think we should talk to this and create a concept how to test this package. It seems a little bit complex.

File: .phpunit.result.cache [hunk 3]
```diff
@@ -0,0 +1 @@
+{"version":1,"defects":{"\/Users\/agaal\/Projekte\/gipfel.dev\/laravel-beyond\/tests\/ExampleTest.php::creates":4,"\/Users\/agaal\/Projekte\/gipfel.dev\/laravel-beyond\/tests\/CommandsTest.php::make":4,"\/Users\/agaal\/Projekte\/gipfel.dev\/laravel-beyond\/tests\/ResolversTest.php::resolve":4},"times":{"\/Users\/agaal\/Projekte\/gipfel.dev\/laravel-beyond\/tests\/ExampleTest.php::example":0.001,"\/Users\/agaal\/Projekte\/gipfel.dev\/laravel-beyond\/tests\/ExampleTest.php::creates":0,"\/Users\/agaal\/Projekte\/gipfel.dev\/laravel-beyond\/tests\/CreateFileFromStubTest.php::creates":0,"\/Users\/agaal\/Projekte\/gipfel.dev\/laravel-beyond\/tests\/CommandsTest.php::make":0.005,"\/Users\/agaal\/Projekte\/gipfel.dev\/laravel-beyond\/tests\/ResolversTest.php::it":0.001,"\/Users\/agaal\/Projekte\/gipfel.dev\/laravel-beyond\/tests\/ResolversTest.php::resolve":0}}
```
[the remark this answers is not in the document]
Comment (alexgaal) [reply .phpunit.result.cache]: already did

File: composer.json [hunk 4]
```diff
@@ -37,5 +37,8 @@
                 "Regnerisch\\LaravelBeyond\\LaravelBeyondServiceProvider"
             ]
         }
+    },
+    "require-dev": {
+        "pestphp/pest": "^1.21"
```
[the remark this answers is not in the document]
Comment (alexgaal) [reply composer.json:42]: I wanted to try pest, that's it.

File: tests/ResolversTest.php [hunk 5]
```diff
@@ -0,0 +1,90 @@
+<?php
+
+test('resolve app name correctly', function () {
+    $appName = 'App';
+    $moduleName = 'Testing';
+    $className = 'TestClass';
+
+    $appNameSchema = "{$appName}/{$moduleName}/{$className}";
```
[the remark this answers is not in the document]
Comment (alexgaal) [reply tests/ResolversTest.php:8]: ?

**Pull request closed without merging**